content/en/docs/collector/custom-collector.md

```diff
@@ -169,6 +165,16 @@ receivers:
       go.opentelemetry.io/collector/receiver/otlpreceiver {{% version-from-registry collector-receiver-otlp %}}
 ```
 
+{{% alert color="primary" title="Tip" %}}
+
+To find components that you can add to your collector component, take a look
```

Comment:
```suggestion
To find components that you can add to your Collector component, browse
```

To your Collector component or Collector components?

Reply:
custom Collector probably better?

Reply:
```suggestion
To find components that you can add to your custom Collector, take a look
```